src/Microsoft.AspNet.Server.Kestrel/Http/SocketOutput.cs

```diff
@@ -135,7 +135,15 @@ public void End(ProduceEndType endType)
 
         private void ScheduleWrite()
         {
-            _thread.Post(_this => _this.WriteAllPending(), this);
+            // Don't post write to closed socket
+            if (!_socket.IsClosed)
```

Comment:
The issue is the loop is the gone hence all the calls erroring are actually `uv_async_send` so its post that fails. However the socket has been successfully closed on the server; so a check to see if the socket is closed fixes it; rather than checking if the loop is valid.

It occurs when `socketShutdownSend` has happened; the client closes the socket, the server picks this up and then the close is kicked off; so when there is a slight delay between `socketShutdownSend` and `socketDisconnect` while the loop has terminated because its been asked to and all the sockets are closed.

Its potentially only in a full shutdown scenarios (e.g. the tests or controlled app recycle)